server/entry/util.ts

```diff
@@ -18,7 +18,7 @@ export type EntryResponse = {
  * @param {Date} date - A date object
  * @returns {string} - formatted date as string
  */
-const formatDate = (date: Date): string => moment(date).format('YYYY/MM/DD');
+const formatDate = (date: Date): string => moment(date).format('YYYY-MM-DD');
```

Comment:
I'm assuming this is the only format `<input type="date">` takes?

Reply:
yeah I believe so. There might be a way to change it? This is definitely the default though.

Reply:
At least in the form we need to represent it this way, but of course you can display it however you like in the table.